release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2478,6 +2478,155 @@ index a90ab14ff..f7f6f1a71 100644
+
+* The `SiteConfig` disk partition definition fails when applied to multiple nodes in a cluster. When a `SiteConfig` CR is used to provision a compact cluster, creating a valid `diskPartition` config on multiple nodes fails with a Kustomize plug-in error. (link:https://issues.redhat.com/browse/OCPBUGSM-44403[*OCPBUGSM-44403*])
+
+* When you patch the `ArgoCD` resource with the ZTP container, the patch points to a tag which follows the latest container version for that release version. If you want to pin the ZTP container to a specific version within the release, the patch file `argocd-openshift-gitops-patch.json` should be updated to point to the specific version. (link:https://issues.redhat.com/browse/OCPBUGSM-44261[*OCPBUGSM-44261*])
+
+*  Applying a `BMCEventSubscription` CR fails to create a Redfish events subscription. After the subscription YAML file is created and applied, no active Redfish subscription is visible. To workaround this issue, run the following command substituting appropriate BMC parameters to call the API directly and create the subscription:
++
+[source,terminal]
+----
+$ curl -X POST -i --insecure -u "<BMC_USERNAME>:<BMC_PASSWORD>" https://<BMC_IP>/redfish/v1/EventService/Subscriptions \
+-H 'Content-Type: application/json' \
+--data-raw '{
+"Protocol": "Redfish",
+"Context": "any string is valid",
+"Destination": "https://hw-event-proxy-openshift-hw-events.apps.example.com/webhook",
+}'
+----
++
+(link:https://issues.redhat.com/browse/OCPBUGSM-43707[*OCPBUGSM-43707*])
+
+* Using the GitOps ZTP pipeline to install a {sno} cluster in a disconnected environment, there should be two `CatalogSource` CRs applied in the cluster. One of the `CatalogSource` CRs gets deleted following multiple node reboots. As a workaround, you can change the default names, such as `certified-operators` and `redhat-operators`, of the catalog sources. (link:https://issues.redhat.com/browse/OCPBUGSM-46245[*OCPBUGSM-46245*])
+
+* Several clusters fail to update during scale testing. After starting {product-title} 4.9.26 update with the Telecom vDU configuration applied, a cluster update to 4.10.13 initiated with the `ClusterVersion` CR fails at 15% and waits for cluster Operators updating to the target version. (link:https://issues.redhat.com/browse/OCPBUGSM-44655[*OCPBUGSM-44655*])
+
+* Spoke cluster deployment on bare-metal hosts using the converged workflow is blocked at inspection. An {rh-rhacm} 4.11 hub cluster using the `assisted-service` API gets stuck in an `inspecting` state, and eventually fails with an `Introspection` timeout. (link:https://issues.redhat.com/browse/OCPBUGSM-45428[*OCPBUGSM-45428*])
```

Comment:
OCPBUGSM-45428 was already fixed in bz https://bugzilla.redhat.com/show_bug.cgi?id=2096460 - I'm not sure why the OCP bug is still open but I will verify it. This entry can be removed